src/client/components/pages/Schedule/SchedulePage.tsx

```diff
@@ -216,9 +231,12 @@ const SchedulePage: FunctionComponent = () => {
 
   return (
     <>
-      <VerticalSpace>
+      <div aria-label="course filter buttons">
+        <span id="Course Filter Button">show/hide catalog prefixes</span>
         {prefixes.map((prefixObj) => (
           <PrefixButton
+            isPrefixActive={isPrefixActive}
```

Comment:
Rather than passing `isPrefixActive` as a function, I'd suggest making it a boolean prop and just calling it with `prefix` here as use time. We can also change the name to just `isActive` to avoid confusion with the function defined above.

```tsx
  isActive={isPrefixActive(prefixObj.prefix)}
``` 

I don't think this makes too much of a different technically, but it does make the component definition easier when you don't need as many embedded calls to `isPrefixActive(prefix)`...